package.json

```diff
@@ -17,58 +17,68 @@
   },
   "scripts": {
     "build": "yarn build:all",
-    "build:all": "yarn build:packages && yarn build:refapps && yarn download:refapps && yarn build:docker",
+    "build:all": "yarn build:packages && yarn build:refapps && yarn build:docker",
     "build:all-packages": "yarn build:packages && yarn build:refapps",
+
     "prebuild:refapps": "yarn download:refapps",
-    "build:refapps": "LOCAL_PACKAGES=true yarn build:refapps:node && LOCAL_PACKAGES=true yarn build:refapps:python",
-    "build:refapps:python": "lerna run build:refapps:python",
-    "build:refapps:node": "lerna run build:refapps:node",
-    "prebuild:refapps:node": "NO_WORKSPACE=true scripts/build-all.js --dirs packages/reference-apps/* --config-name=tsconfig.json --copy-dist",
-    "postbuild:refapps": "LOCAL_PACKAGES=true lerna run postbuild:refapps",
-    "build:packages": "scripts/build-all.js --dirs packages/* --config-name=tsconfig.build.json --copy-dist",
-    "prebuild:packages": "LOCAL_PACKAGES=true lerna run build:only && LOCAL_PACKAGES=true lerna run postbuild",
-    "build:docker": "lerna run build:docker",
+    "build:refapps": "yarn build:refapps:node",
+    "postbuild:refapps": "scripts/run-script.js -v -w refapps postbuild:refapps",
+    "prebuild:refapps:node": "scripts/build-all.js -v -w refapps --outdir=dist/reference-apps --no-install --no-distws --bundle",
+    "build:refapps:node": "scripts/run-script.js -v -w refapps build:refapps:node",
+
+    "prebuild:packages": "scripts/run-script.js -v -w packages build:only",
+    "build:packages": "scripts/build-all.js -v -w packages --ts-config tsconfig.build.json",
+
+    "build:docker": "scripts/run-script.js -w packages build:docker",
+
     "build:readme": "node ./scripts/mk-readme",
-    "build:docs": "lerna run build:docs",
-    "build:all-docs": "node ./scripts/mk-readme && lerna run build:docs",
+    "build:docs": "scripts/run-script.js -w packages build:docs",
+    "build:all-docs": "yarn build:readme && yarn build:docs",
+
+    "pack:pub": "yarn clean:root && FLAT_PACKAGES=true MAKE_PUBLIC=true NO_INSTALL=true scripts/build-all.js -w '!bdd' --ts-config tsconfig.build.json",
+    "publish:dist": "find dist/ -iname package.json -not -wholename '*node_modules*' -execdir yarn publish --no-git-tag-version --non-interactive \\;",
```

Comment:
I'm trying to understand the relationship between these two and `build:*`. `build` happens inside each package, and these moive them to global `dist`, is that right? If so, does this mean that to run STH from compiled JS (e.g. with `yarn start`) I have to run these commands first? (meaning `build:packages` will not be enough?)

Also, should these two be separate? It seems to me that they'll always be ran together anyway.